stable/mongodb/templates/statefulset-primary-rs.yaml

```diff
@@ -265,7 +266,17 @@ spec:
           configMap:
             name: {{ template "mongodb.fullname" . }}
         {{- end }}
-{{- if .Values.persistence.enabled }}
+        {{- if not .Values.persisntence.enabled }}
```

Comment:
There is another typo here `persisntence` should be `persistence`.

Reply:
Thanks, fixed.